doc/vim-lsp.txt

```diff
@@ -1202,6 +1203,16 @@ only possible by adding to the already typed prefix (even if you're using the
 `contains` filter). If you'd like to retrigger the filtering, you will have to
 press CTRL-X CTRL-O again.
 
+sort                                              *vim-lsp-completion-sort*
+
+You can sort the completion items returned from the server by using the `sort` key in the server info's `config` |dict|.
```

Comment:
This line is too long. Please take care about line width. Other lines are also exceeding the width imit.

Reply:
Thank you for pointed out!
Fixed!